### ISSUE TITLE
no polyaxon_ip and polyaxon command not found

### ISSUE THREAD

**Issue by ziyitony** [polyaxon/polyaxon#137]
### Unclear documentation
`$ helm install polyaxon/polyaxon \
--name=polyaxon \
--namespace=polyaxon \
-f config.yml`

https://docs.polyaxon.com/installation/deploy_polyaxon/

### Missing Documentation
Thanks for this good repo. I followed the installation steps in the homepage but failed.

When I deploy polyaxon I always failed, neither use default values by
 `$ helm install polyaxon/polyaxon --name=polyaxon --namespace=polyaxon ` 
or do a little change in config file
`rbac:
  enabled: false
ingress:
  enabled: false
serviceType: LoadBalancer` 
and then 
`$ helm install polyaxon/polyaxon \
--name=polyaxon \
--namespace=polyaxon \
-f config.yml`

, I get output but can't going on. **### no polyaxon_ip and polyaxon command not found**.
Since I am new from k8s and polyaxon, I don't know if it is because of my mistake in installing k8s. Thanks for your help.

![version](https://user-images.githubusercontent.com/18106851/42512413-1f935d04-8487-11e8-905d-213f60a465fa.png)

![anything ready](https://user-images.githubusercontent.com/18106851/42512865-365d1c2c-8488-11e8-9c95-ae5e3228ec8b.png)

![notes1](https://user-images.githubusercontent.com/18106851/42512873-3b7906ee-8488-11e8-8211-0e14ac4ce78b.png)

![notes2](https://user-images.githubusercontent.com/18106851/42512878-3e4c9926-8488-11e8-95b6-5a8bfad94dac.png)

![stucked](https://user-images.githubusercontent.com/18106851/42512886-40c7e1f6-8488-11e8-91f2-ce4f89b6193b.png)

![no ip](https://user-images.githubusercontent.com/18106851/42513193-e8fddfba-8488-11e8-9d96-3c4e2c3c6f78.png)

**Comment by mouradmourafiq** [comment]
@ziyitony Regarding the error: `polyaxon: command not found`, you need to [install the cli](https://docs.polyaxon.com/installation/install_polyaxon_cli/#install).

For the IP error, are using Minikube? if so then the ip address is the ip of of your cluster, normally if you deploy with `isMinikube: true` the NOTES will be different, `minikube ip`

**Comment by ziyitony** [comment]
@mouradmourafiq  your help is verrrrrrrrrrry kind! Yes, by polyaxon-cli, I can use polyaxon command now.
But I deploy it in k8s clusters set up by kubeadm, not minikube.

**Comment by ziyitony** [comment]
I destroyed the vm and reinstalled Ubuntu 16.0, and deployed k8s by kubeadm.
This time I used canal pod network add-on rather than flannel.
And I stop ubuntu filewall and my network security rule of vm.
And deployed polyaxon by
 `helm install polyaxon/polyaxon --name=polyaxon --namespace=polyaxon`
It worked now. Don't know which step works, hope it helpful for other user.

![it works](https://user-images.githubusercontent.com/18106851/42573254-2ab9aef0-854e-11e8-960d-e6a3c5edbd5f.png)
